lua/starfall/libs_cl/vgui.lua

```diff
@@ -0,0 +1,2782 @@
+-- Global to all starfalls
+local checkluatype = SF.CheckLuaType
+local registerprivilege = SF.Permissions.registerPrivilege
+haspermission = SF.Permissions.hasAccess
+local function checkoptional(val, chk)

```

Comment:
You define this function but don't use it for all cases, either use it for all or just don't define the function at all (since the rest of the codebase doesn't define a function, either)

Reply:
Removed